lum-network/assetlist.json

```
@@ -0,0 +1,28 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_id": "lum-network-1",
+    "assets": [
+        {
+            "description": "Native token of the Lum Network",
+            "denom_units": [
+                {
+                    "denom": "ulum",
+                    "exponent": 0
+                },
+                {
+                    "denom": "lum",
+                    "exponent": 6
+                }
+            ],
+            "base": "ulum",
+            "name": "Lum",
+            "display": "lum",
+            "symbol": "LUM",
+            "logo_URIs": {
+                "png": "https://raw.githubusercontent.com/lum-network/mainnet/master/assets/lum.png",
+                "svg": "https://raw.githubusercontent.com/lum-network/mainnet/master/assets/lum.svg"
+            },
+            "coingecko_id": "lum-network"
```

```suggestion
            }
```

Please don't include unless actually listed on coingecko.  Feel free to update when you are.